src/tools/info/tl_info.c

```diff
@@ -125,6 +120,12 @@ static void print_iface_info(uct_worker_h worker, uct_md_h md,
         .stats_root            = ucs_stats_get_root(),
         .rx_headroom           = 0
     };
+    char buf[200]                   = {0};
```

Comment:
should it be aligned with the declarations above?

Reply:
looks like it is aligned with line 111, but i'd just use single space here